.github/workflows/ci-notify-slack.yml

```diff
@@ -18,32 +18,22 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
             const { owner, repo, number: issue_number } = context.issue;
-            const team_reviewers = context.payload.pull_request.requested_teams.map(team => team.name);
+            const team_reviewers = context.payload.requested_team.name;
             return team_reviewers;
       - name: Lookup Slack channel
         id: lookup
         run: |
-          TEAM_REVIEWERS=$(echo '${{ steps.get-reviewers.outputs.result }}' | jq -c '.[]')
-          CHANNELS=""
-          for TEAM in $TEAM_REVIEWERS; do
-            CHANNEL=$(jq -r ".$TEAM" .github/workflows/team-channels.json)
-            if [ "$CHANNEL" != "null" ]; then
-              if [ -z "$CHANNELS" ]; then
-                CHANNELS="$CHANNEL"
-              else
-                CHANNELS="$CHANNELS,$CHANNEL"
-              fi
-            fi
-          done
-          echo "channels=${CHANNELS}" >> $GITHUB_OUTPUT
+          TEAM=${{ steps.get-reviewers.outputs.result }}
+          CHANNEL=$(jq -r ".$TEAM" .github/workflows/team-channels.json)
```

Comment:
We should handle the case where the team doesn't have a channel assigned yet. The following step would fail in that case I assume.